packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -312,17 +314,30 @@ export class TopNavigation {
             <header role="banner">
               <div class="top-panel-container">
                 <div class="app-details-container">
-                  {hasTitle && (
-                    <a class="title-link" href={this.href}>
+                  {isSlotUsed(this.el, "router-app-title") ? (
+                    <div class="title-link">
                       {this.hasAppIcon && (
                         <div class="app-icon-container" aria-hidden="true">
                           <slot name="app-icon" />
                         </div>
                       )}
                       <ic-typography variant={appTitleVariant}>
-                        <h1 class="title-wrap">{this.appTitle}</h1>
+                        <slot name="router-app-title"></slot>
```

Comment:
Should we consider just naming the slot something like "app-title". I mention so we can be consistent with our naming and dropping the word "router". For example, Page header uses "navigation-item" slot, same as side navigation for the navigation items. Footer also uses just "link" for the same use case.